### PR TITLE
Add a script to generate a list of deleted users uids

### PR DESCRIPTION
**Pull request by simonpoole**
No idea if this works and there don't seem to be tests for anything similar. Lets see what CI says.

### CODE REVIEW

File: cookbooks/planet/templates/default/users-deleted.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# DO NOT EDIT - This file is being maintained by Chef
+
+T=$(mktemp -d -t -p /var/tmp users.XXXXXXXXXX)
+
+# use the same as for the users-agreed template
+export PGPASSFILE=/etc/replication/users-agreed.conf
+
+echo "# user IDs of deleted users. " > $T/users_deleted
+psql -h <%= node[:web][:readonly_database_host] %> -U planetdiff -t -c "select id from users where status='deleted' by id asc" openstreetmap >> $T/users_deleted
```
Comment (tomhughes) [inline cookbooks/planet/templates/default/users-deleted.erb]: I think you meant "order by id asc" here.

Comment (simonpoole) [reply cookbooks/planet/templates/default/users-deleted.erb]: yes ... copy paste error ...

**Comment by mmd-osm** [comment]
@simonpoole : I believe this change was somehow GDPR related, but I couldn't figure out why the SELECT statement didn't include `changesets_count > 0` as a condition. It would be a prerequisite for a user to show up in any public facing planet dump or diff file.

Nowadays, the users_deleted.txt file is fairly large, and most users are likely spam accounts that never contributed anything.

**Comment by simonpoole** [comment]
While having a non-zero number of changesets is a prerequisite for tuning up in the data dumps, it is conceivable that the user in question used the account to authenticate in a 3rd party service, so I'm not quite sure that they could just be left away.

**Comment by mmd-osm** [comment]
> maybe we should simply have two lists

Yes, that sounds like a good idea.

**Comment by mmd-osm** [comment]
Any idea what would be a good filename for the second list? Something along the lines of _active_users_deleted,_ or  _contributors_deleted_, maybe?

**Pull request merged**